Tests/CMakeLists.txt

```diff
@@ -7,27 +7,31 @@ Licensed under Apache License v2.0 with Runtime Library Exception
 See https://swift.org/LICENSE.txt for license information
 #]]
 
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin) # FIXME Hook this up on Darwin
```

Comment:
The reason that this was guarded against Darwin because dispatch and Foundation are provided by the system.  Is the intent to build against a non-system copy of dispatch and swift-corelibs-foundation.

Reply:
The issue here is that CMake isn't set up to allow importing XCTest in Swift projects on Darwin -- the stock FindXCTest.cmake thingy finds the Objective-C framework, but it doesn't configure the overlay paths. 

I didn't have time to figure this out, and I think it's preferable to ship without a test target on Darwin rather than shipping with a broken one.